### ISSUE TITLE
Debugger: Add Support for PDB and LLDB

### ISSUE THREAD

**Issue by DevynCJohnson** [geany/geany-plugins#886]
This may already be on the developer's to-do list, but I think it would be very helpful if the Debugger add-on had support for PDB (Python Debugger) and LLDB (LLVM Debugger).

**Comment by KhazAkar** [comment]
Yes, that would be a great addition and would help a lot to have disassembly view :)

**Comment by sollted** [comment]
yes please

**Comment by elextr** [comment]
All the patches to debugger in at least the last six years are bandaids to keep it running, and the "maintainer" has not contributed for ten years AFAICT.  So "somebody" needs to take over maintenance, and then make the changes.
